packages/test/test-end-to-end-tests/.vscode/launch.json

```diff
@@ -0,0 +1,235 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+	"version": "0.2.0",
+	"configurations": [
```

Comment:
I've personally never really had an issue running e2e tests with different drivers or anything. I'm willing to try this out.

As for the performance hit for legacy tests, I have some draft that I think makes it way faster, but I haven't had time to get it production ready: https://github.com/microsoft/FluidFramework/pull/21881

Reply:
we definitely can do better perf-wise in the install step. The perf hit that this comment / option refers to is from a different place though: VSCode takes quite a while to load all of the `outFiles` in our monorepo if we're not careful. You can see this by using `"trace": true` in one of the launch targets and playing around with https://microsoft.github.io/vscode-pwa-analyzer/index.html.

Unsurprisingly, it's therefore a noticeable perf hit to tell vscode to load and make sure breakpoints get set in all of the extra versions of Fluid we install for our compat test matrix, especially when debugging e2e tests often doesn't require it.